packages/cactus-plugin-bungee/tsconfig.json

```diff
@@ -0,0 +1,41 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "composite": true,
+    "outDir": "./dist/lib/", /* Redirect output structure to the directory. */
+    "declarationDir": "dist/lib",
+    "resolveJsonModule": true,
+    "allowJs": false,
+    "skipLibCheck": true,
```

Comment:
@eduv09 Does this have to be set this way or could it be set to `false`? If it is this way for a specific reason then let's talk about it and try to fix it together os that it doesn't have to be disabled (the lib checking)

Reply:
@eduv09 Also investigated this one and this is fine as-is, resolving.